torchdynamo/variables/builder.py

```diff
@@ -238,7 +238,7 @@ def _wrap(self, value):
             return SkipFilesVariable(
                 value, guards=make_guards(GuardBuilder.FUNCTION_MATCH)
             )
-        elif istype(value, type):
+        elif inspect.isclass(value):
```

Comment:
any ideas on how to test the ABCMeta fix?  I wasn't able to think of a simple unit test that would exercise this bug

Reply:
Hrm, I think you test should exercise this... does it not?

Reply:
i couldn't get the test to fail with `istype(value, type)` despite trying a few different things with super.  I'll have another go at it.